python/build-wheel-manylinux2014.sh

```diff
@@ -35,14 +35,14 @@ echo "java_bin path $java_bin"
 java_home=${java_bin%jre/bin/java}
 export JAVA_HOME="$java_home"
 
-/ray/ci/env/install-bazel.sh
+$WORK_DIR/ci/env/install-bazel.sh
 # Put bazel into the PATH if building Bazel from source
 # export PATH=/root/bazel-3.2.0/output:$PATH:/root/bin
 
 # If converting down to manylinux2010, the following configuration should
 # be set for bazel
 #echo "build --config=manylinux2010" >> /root/.bazelrc
-echo "build --incompatible_linkopts_to_linklibs" >> /root/.bazelrc
+echo "build --incompatible_linkopts_to_linklibs --jobs=4" >> /root/.bazelrc
```

Comment:
What does this do and why do we set it for `ray`? (A line of comment clarifying this would be useful.

Thanks

Reply:
Cool. Will do that. It is to limit the number of parallel jobs that it runs. the CCI was going OOM and had to move to 2xlarge. So just limited the number of jobs that run.